x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/components/crawl_requests_table.tsx

```diff
@@ -45,6 +40,16 @@ const columns: Array<EuiTableFieldDataColumnType<CrawlEvent>> = [
       <CustomFormattedTimestamp timestamp={createdAt} />
     ),
   },
+  {
+    field: 'type',
+    name: i18n.translate(
+      'xpack.enterpriseSearch.appSearch.crawler.crawlRequestsTable.column.crawlType',
+      {
+        defaultMessage: 'Crawl Type',
```

Comment:
According to Kibana writing guidelines, this should probably be `Crawl type`.

Reply:
Fixed in 98f2f26